src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -1985,6 +1989,61 @@ void canOverrideRenewalAfterTwoDeclaredLostAndRefunds() {
+    final UUID servicePointId = servicePointsFixture.cd6().getId();
+    feeFineOwnerFixture.ownerForServicePoint(servicePointId);
+    useLostItemPolicy(lostItemFeePoliciesFixture.chargeFeeWithActualCost().getId());
+    IndividualResource loan = checkOutFixture.checkOutByBarcode(item, jessica,
+      ZonedDateTime.of(2018, 4, 21, 11, 21, 43, 0, UTC));
+    declareLostFixtures.declareItemLost(loan.getJson());
+
+    postLostItemActualCostAccountForLoan(loan, jessica, 10.0);
+
+    assertThat(feeFineActionsClient.getAll(), hasSize(2));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item fee (actual cost)", "Open"), allOf(
+      hasJsonPath("amount", 10.0), hasJsonPath("remaining", 10.0)));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item processing fee", "Open"), allOf(
+      hasJsonPath("amount", 5.0), hasJsonPath("remaining", 5.0)));
+
+    feeFineAccountFixture.payLostItemActualCostFee(loan.getId(), 3.0);
+    feeFineAccountFixture.payLostItemProcessingFee(loan.getId(), 3.0);
+
+    final OkapiHeaders okapiHeaders = buildOkapiHeadersWithPermissions(
+      OVERRIDE_RENEWAL_BLOCK_PERMISSION);
+    JsonObject renewedLoan = loansFixture.renewLoan(
+      buildRenewByBarcodeRequestWithRenewalBlockOverride(item, jessica, servicePointId.toString()),
+      okapiHeaders).getJson();
+
+    assertThat(renewedLoan.getString("action"), is(RENEWED_THROUGH_OVERRIDE));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item fee (actual cost)", "Closed"), allOf(
+      hasJsonPath("amount", 10.0), hasJsonPath("remaining", 0.0)));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item processing fee", "Closed"), allOf(
+      hasJsonPath("amount", 5.0), hasJsonPath("remaining", 0.0)));
+
+    declareLostFixtures.declareItemLost(renewedLoan);
+    postLostItemActualCostAccountForLoan(loan, jessica, 10.0);
+
+    assertThat(getAccountForLoan(loan.getId(), "Lost item fee (actual cost)", "Open"), allOf(
+      hasJsonPath("amount", 10.0), hasJsonPath("remaining", 10.0)));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item processing fee", "Open"), allOf(
+      hasJsonPath("amount", 5.0), hasJsonPath("remaining", 5.0)));
+
+    feeFineAccountFixture.payLostItemActualCostFee(loan.getId(), 3.0);
+    feeFineAccountFixture.payLostItemProcessingFee(loan.getId(), 3.0);
+
+    JsonObject secondRenewedLoan = loansFixture.renewLoan(
+      buildRenewByBarcodeRequestWithRenewalBlockOverride(item, jessica, servicePointId.toString()),
+      okapiHeaders).getJson();
+    assertThat(secondRenewedLoan.getString("action"), is(RENEWED_THROUGH_OVERRIDE));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item fee (actual cost)", "Closed"), allOf(
+      hasJsonPath("amount", 10.0), hasJsonPath("remaining", 0.0)));
+    assertThat(getAccountForLoan(loan.getId(), "Lost item processing fee", "Closed"), allOf(
+      hasJsonPath("amount", 5.0), hasJsonPath("remaining", 0.0)));
+  }
+
```

Comment:
I would suggest moving these tests to OverrideRenewDeclaredLostItemTest. We have some common code that can be extracted into the general parent RefundDeclaredLostFeesTestBase